migrations/V10__init_schema.sql

```diff
@@ -0,0 +1,47 @@
+CREATE TABLE header
```

Comment:
Don't want to deviate you too far off course, but I'm wondering if about your thoughts on using an ORM? I personally find them a lot less daunting than modifying raw sql. In any case, this schema feels small enough to be manageable without an ORM.

Reply:
Not a bad idea. I don't think we get the full leverage of using an ORM since we aren't really doing complicated things with the query results. We're mainly just loading a bunch of data and passing it straight through to the client. But it would handle a lot of the boilerplate parsing and query generation. I'll make an issue and we can do this as a follow-up without changing the interface.

Reply:
https://github.com/EspressoSystems/hotshot-query-service/issues/270